MonoZelda/Enemies/EnemyStateMachine.cs

```diff
@@ -170,7 +170,7 @@ public Point Update(Enemy enemy, Point position)
                 {
                     enemySpriteDict.Enabled = false;
                     enemy.Alive = false;
-                    if (enemy.GetType() != typeof(Keese) && enemy.GetType() != typeof(Gel))
+                    if (enemy.GetType() != typeof(Keese) && enemy.GetType() != typeof(Gel) && enemy.GetType() != typeof(Zol))
```

Comment:
C# has an `is` operator. You could do `!(enemy is Zol)`. `enemy is not Zol` is possible in current version of C#. I don't know if we have that.